### PR TITLE
feat(backuptarget): add cleanup all mounts grpc function

### PR DESCRIPTION
**Pull request by ChanYiLin**
ref: [longhorn/longhorn#5741](https://github.com/longhorn/longhorn/issues/5741)

unmount the mountpoints in instance-manager in every controller when Target is unset

~Blocked by: https://github.com/longhorn/longhorn-instance-manager/pull/305~

### CODE REVIEW

**Comment by ChanYiLin** [comment]
> This pull request is now in conflicts. Could you fix it @ChanYiLin? 🙏

conflicts resolved

**Review by innobead — DISMISSED** [review]

File: engineapi/proxy_backup.go [hunk 1]
```diff
@@ -87,3 +87,8 @@ func (p *Proxy) BackupRestoreStatus(e *longhorn.Engine) (status map[string]*long
 	}
 	return status, nil
 }
+
+func (p *Proxy) CleanupBackupMountPoints() (err error) {
+	err = p.grpcClient.CleanupBackupMountPoints()
+	return err
```
Comment (innobead) [inline engineapi/proxy_backup.go:93]: nit: 

```go
	return p.grpcClient.CleanupBackupMountPoints()
```

**Comment by innobead** [comment]
@ChanYiLin next time, can squash the commit first. In this PR, some commits are just transient.

File: engineapi/proxy_backup.go (hunk 1 above)
Comment (ChanYiLin) [reply engineapi/proxy_backup.go]: Fixed, thanks!

**Comment by ChanYiLin** [comment]
> @ChanYiLin next time, can squash the commit first. In this PR, some commits are just transient.

Hi @innobead , rebase the commits, thanks!

**Review by innobead — APPROVED** [review]

**Comment by ChanYiLin** [comment]
@mergify backport v1.5.x

**Pull request merged**